index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+We are Microvers Programers
+<header id="header">
+  <img src="https://cdnprod.mafretailproxy.com/sys-master-root/h95/ha8/14539669733406/404925_main.jpg_1700Wx1700H
+  "  id="header-img" width="100" height="200"/>
+  <nav id="nav-bar"> 
+    <button class="nav-link" type="button">Click Me!</button>
+    <button class="nav-link" type="button">Click Me!</button>
+    <button class="nav-link" type="button">Click Me!</button>
+  </nav>
```

Comment:
The button must be in a grid container. Yours works fine now, but the requirement asks you to use a grid display because it's possible that in the future, when we adapt to desktop display, grid display will be important. :Cactus: . You're doing good, be patient.

Reply:
I will fix this